src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/RemoveUnnecessaryLineContinuationCodeCleanupProvider.vb

```diff
@@ -20,6 +20,12 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
         End Property
 
         Public Async Function CleanupAsync(document As Document, spans As IEnumerable(Of TextSpan), Optional cancellationToken As CancellationToken = Nothing) As Task(Of Document) Implements ICodeCleanupProvider.CleanupAsync
+            ' Is this VB 9? If so, we shouldn't remove line continuations because implicit line continuation was introduced in VB 10.
+            Dim parseOptions = TryCast(document.Project.ParseOptions, VisualBasicParseOptions)
```

Comment:
Reasonable since it's in the stabilization branch, but as long as the language of the document is VB (which I'd presume is the case since we're already in the language-specific service), this cast can't fail.